### PR TITLE
Add VGLZ1 VERI Gold 1 Ounce token

### PR DESCRIPTION
**Pull request by therealMikkoK**
#1568

### CODE REVIEW

File: tokens/0xd1ec43b9d44338a4e74f3f41cd1b86254cbaa874.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+addr: '0xd1ec43b9d44338a4e74f3f41cd1b86254cbaa874'
+decimals: 18
+description: >- Tokenized 1 Troy Ounce Gold Bar, Redeemable in Precious Metal (Gold)
```
Comment (freeatnet) [inline tokens/0xd1ec43b9d44338a4e74f3f41cd1b86254cbaa874.yaml]: Description should probably include a bit more info. Eg., see the description in the original request: https://github.com/forkdelta/tokenbase/issues/1568#issue-349772441

On the technical side, text after `>-` should be placed on the next line, indented with 2 spaces, like so:
```yaml
description: >-
  VGLZ1 is a sub-token of VERI (VERITASEUM) Ecosystem.
  
  VGLZ1 is a digital token asset representing 1oz physical fine .999 gold that can be exchanged
  and then redeemed via the Veritaseum website, or VeADIR Dapp after KYC is provided.
```
Single line breaks will be converted to a space, two line breaks (or two spaces followed by a line break) will produce a new line (like Reddit comments). (see: https://github.com/forkdelta/tokenbase/blob/master/README.md#description)

File: tokens/0xd1ec43b9d44338a4e74f3f41cd1b86254cbaa874.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  # The following options were found                                                                       #
+  # You may need to edit them and you MUST delete commented lines                                          #
+  # From the website: description meta tag                                                                 #
+  Veritaseum enables software-driven P2P capital markets without brokerages, banks or traditional exchanges
```
Comment (freeatnet) [inline tokens/0xd1ec43b9d44338a4e74f3f41cd1b86254cbaa874.yaml]: The template comments (lines that start with `#`) and the auto-generated description should be deleted.

File: tokens/0xd1ec43b9d44338a4e74f3f41cd1b86254cbaa874.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+links:
+- Facebook: https://www.facebook.com/reggiemiddletonfintech/?fref=ts
+- Telegram: https://t.me/joinchat/G_22OhA7UgN6b-7hTSFSTA
+- Twitter: https://twitter.com/reggiemiddleton
+- Website: https://veritas.veritaseum.com/
+- YouTube: https://www.youtube.com/channel/UCjVEF7ABJwyzw_8pb5frVmg
+- Blog: https://veritas.veritaseum.com/ve-blog
+- Reddit: https://www.reddit.com/r/VeritaseumCommunity/
+
+
```
Comment (freeatnet) [inline tokens/0xd1ec43b9d44338a4e74f3f41cd1b86254cbaa874.yaml]: Extra white space here should be deleted.

File: tokens/0xd1ec43b9d44338a4e74f3f41cd1b86254cbaa874.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  # From the website: description meta tag                                                                 #
+  Veritaseum enables software-driven P2P capital markets without brokerages, banks or traditional exchanges
+links:
+- Facebook: https://www.facebook.com/reggiemiddletonfintech/?fref=ts
+- Telegram: https://t.me/joinchat/G_22OhA7UgN6b-7hTSFSTA
+- Twitter: https://twitter.com/reggiemiddleton
+- Website: https://veritas.veritaseum.com/
+- YouTube: https://www.youtube.com/channel/UCjVEF7ABJwyzw_8pb5frVmg
+- Blog: https://veritas.veritaseum.com/ve-blog
+- Reddit: https://www.reddit.com/r/VeritaseumCommunity/
```
Comment (freeatnet) [inline tokens/0xd1ec43b9d44338a4e74f3f41cd1b86254cbaa874.yaml]: Is there really no white paper for this token?

Comment (therealMikkoK) [reply tokens/0xd1ec43b9d44338a4e74f3f41cd1b86254cbaa874.yaml]: No white paper.
They do have a working product that operates on the network.
ICO wasn't launched as an investment opportunity, rather the tokens are a prepaid fee.
Earlier version ran on BTC blockhcain, but it was halted due to regulatory uncertainty.
So the secret herbs and spices are not available to see, but we see the VeADIR in action every time an exposure is opened.

File: tokens/0xd1ec43b9d44338a4e74f3f41cd1b86254cbaa874.yaml (hunk 2 above)
Comment (therealMikkoK) [reply tokens/0xd1ec43b9d44338a4e74f3f41cd1b86254cbaa874.yaml]: delete the template comments

File: tokens/0xd1ec43b9d44338a4e74f3f41cd1b86254cbaa874.yaml (hunk 3 above)
Comment (therealMikkoK) [reply tokens/0xd1ec43b9d44338a4e74f3f41cd1b86254cbaa874.yaml]: Extra lines removed

File: tokens/0xd1ec43b9d44338a4e74f3f41cd1b86254cbaa874.yaml (hunk 4 above)
Comment (therealMikkoK) [reply tokens/0xd1ec43b9d44338a4e74f3f41cd1b86254cbaa874.yaml]: Would an introduction on the workings on the company's own website suffice to clearly illustrate what the token does?
The website has ample information on the company.
Also the team is quite impressive.
I have spoken to several personally, if that counts to anything.

Comment (freeatnet) [reply tokens/0xd1ec43b9d44338a4e74f3f41cd1b86254cbaa874.yaml]: Just making sure we link it if one exists.

**Pull request merged**